ow_lander/scripts/path_planning_commander.py

```diff
@@ -215,8 +228,40 @@ def run(self):
     self.guarded_move_srv = rospy.Service(
         'arm/guarded_move', GuardedMove, self.handle_guarded_move)
     print("path_planning_commander has started!")
+
     rospy.spin()
 
+  def stop_arm_if_fault_feedback_cb(self, feedback):
+    """
+    stops arm if arm fault exists during feedback callback
+    """
+    if self.arm_fault: self.trajectory_async_executer.stop()
+
+  def stop_arm_if_fault_active_cb(self):
+    """
+    stops arm if arm fault exists during active callback
+    """
+    if self.arm_fault: self.trajectory_async_executer.stop()
+
+  def return_message(self, action_name, success=None):
+    if not self.arm_fault and success:
+      print(action_name + " activity completed")
+      return success, "Done" 
+    elif not self.arm_fault and success is None:
+      print(action_name + " activity completed")
+      return True, "Done" 
+    else:
+      print(action_name + " activity incomplete")
+      return False,  action_name + " failed"
```

Comment:
An activity can be completed and still fail even when no fault is enabled...
```suggestion
  def return_message(self, action_name, success=True):
    if not self.arm_fault:
      print(action_name + " activity completed")
      return success, "Done"
    else:
      print(action_name + " activity incomplete")
      return False,  action_name + " failed"
```